sql/src/main/java/io/crate/metadata/information/InformationPartitionsTableInfo.java

```diff
@@ -45,6 +45,12 @@
         public static final Reference NUMBER_OF_SHARDS = createRef(Columns.NUMBER_OF_SHARDS, DataTypes.INTEGER);
         public static final Reference NUMBER_OF_REPLICAS = createRef(Columns.NUMBER_OF_REPLICAS, DataTypes.STRING);
         public static final Reference ROUTING_HASH_FUNCTION = createRef(Columns.ROUTING_HASH_FUNCTION, DataTypes.STRING);
+        public static final Reference TABLE_VERSION = createRef(Columns.TABLE_VERSION, DataTypes.OBJECT);
+        public static final Reference TABLE_VERSION_CREATED = createRef(
+            Columns.TABLE_VERSION_CREATED, DataTypes.OBJECT);
+        public static final Reference TABLE_VERSION_UPGRADED = createRef(
+            Columns.TABLE_VERSION_UPGRADED, DataTypes.OBJECT);
```

Comment:
afaik the concrete value columns of that objects are missing, like `crate` string (must be converted from integer), `elasticsearch` string (also converted). otherwise, one could not select `version['created']['crate']` directly